<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Artifact Collector</string>
  </property>
  <widget class="QLabel" name="infoTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>사건 정보</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>일시</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>10</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2023</year>
     <month>9</month>
     <day>8</day>
    </datetime>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::YearSection</enum>
   </property>
  </widget>
  <widget class="QLabel" name="PICLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>담당자</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pic_lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="subjLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>조사 대상</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="subj_lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
